homeassistant/components/amberelectric/sensor.py

```diff
@@ -1,591 +1,247 @@
 """Amber Electric Sensor definitions."""
+
+# TODO Use SEnsor Descriptions to set up all the different types of sensro that are required.
```

Comment:
Do not include to do's in the integration code.

Reply:
That's an outdated commit - That code's been removed.